pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -8,7 +8,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type BannerContent int
+
 const (
+	// not using iota to force a stable value mapping
+	BannerEmpty          BannerContent = 0
+	BannerContactSupport BannerContent = 1
```

Comment:
Actually I picked int here after experiences with the v3 run command Geneva Action. I never remembered what the string should be so I had to go to the code and find it every time.
With 0,1 you can set it quickly, and I agree that if we get to 4 messages it will be problematic. I hope we can do with 1 maybe 2 messages total.

Reply:
@Makdaam how is epereience is going to look like from SRE perspective? Is it going to be:

1. Get cluster object using get kube object geneva action
2. Modify cluster object and update it with put or patch kubernetes object geneva action?

Or are we going to have a separate geneva action for this? If spearate - we can solve the issue of remembering values by providing a dropdown.

Reply:
Ah, I see that you are going to create a powershell script. In this case I think it is still better to have a string: I imagine that the script will be something like "enable/disable banner" and we can put relevant values into the script (so we don't have to remember them), but readability will be better when reading with `oc` or geneva actions.

Reply:
If we are going to have multiple preset messages in the future (which I think would be very useful), then having numbers associated with the messages in the RP might get confusing, especially if we discontinue a message. For example, say we have 5 messages and we discontinue #3... then we want to add a new message. Does the new one go in slot 3, or slot 6? We need to keep in mind that people's local scripts are not always up to date, and we don't want to inadvertently display the wrong message in the console.

Strings would be easier. It could still be easy to use with a script. The script could say something like, "Select the message you want to display in the console" and present a list of options:

0: No message
1: First message text
2: Second message text

The SRE could select an option by (non-permanent, displayed only) number, and then it would be posted. In the background, however, it would be explicitly selecting the desired message by name.

Reply:
Thanks for the points above. Changed to strings.